lib/common.js

```diff
@@ -296,6 +296,122 @@ class TargetItem {
   }
 }
 
+class FieldTarget {
```

Comment:
NOTE: The `FieldTarget` class was just moved from `export.js` to `common.js`.  There were no changes to the actual class definition.